packages/admin/admin-theme/src/componentsTheme/MuiDataGrid.tsx

```diff
@@ -87,18 +88,17 @@ export const getMuiDataGrid: GetMuiComponentTheme<"MuiDataGrid"> = (component, {
             height: "20px",
             marginRight: "10px",
         },
-        panelContent: {
-            [`& .${gridClasses.filterForm}:first-child .${gridClasses.filterFormLinkOperatorInput}`]: {
-                ["@media (max-width: 900px)"]: {
-                    display: "none",
-                },
-            },
-        },
         filterForm: {
-            padding: spacing(4),
+            margin: spacing(7, 4, 4, 4),
+            padding: "7px 5px",
+            borderBottom: `1px solid ${palette.grey[100]}`,
+            "&:last-child": {
+                border: "none",
+            },
 
             [`.${inputLabelClasses.root}`]: {
-                display: "none",
+                transform: "translateY(-22px)",
```

Comment:
This seems odd. Why do we need this?

Reply:
This is necessary to display the labels on top of the inputs correctly